.gitignore

```diff
@@ -129,3 +129,10 @@ venv.bak/
 
 # scons build files
 *.dblite
+
+# user temp files
```

Comment:
@paulharris is there something better to put here?

Reply:
https://github.com/github/gitignore/blob/main/Global/Vim.gitignore

Reply:
That link's stuff looks ok to me (without testing).
Otherwise, my suggestions have worked for me for years, although I don't have tags and I put my undo files elsewhere, so the link would probably work for a wider range of users.

Reply:
```suggestion
# vim temp files
```